### PR TITLE
fix-Alphabetical order in Trusted Merchants

### LINKED ISSUE #323: Alphabetical order is missing in Trusted Merchants
**Summary:**
.
Alphabetical order is missing in Trusted Merchants

**Device and Android version:**

Device Real Me 2
Android Version 8.1.0

**Screenshots:**

videotogif_2019 02 17_09 11 14

### PR DESCRIPTION
**Pull request by shaliniv16**
Fixes #323 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by shaliniv16** [comment]
@luckyman20 please review.

**Comment by RajaVamsi11** [comment]
@shaliniv16 As this is a UI change, can you please add screenshot or GIF.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/merchants/adapter/MerchantsAdapter.java [hunk 1]
```diff
@@ -58,6 +60,12 @@ public int getItemCount() {
     }
 
     public void setData(List<SavingsWithAssociations> mMerchantsList) {
+        Collections.sort(mMerchantsList, new Comparator<SavingsWithAssociations>() {
```
Comment (naman14) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/merchants/adapter/MerchantsAdapter.java]: Business logic shouldn't be in adapter, sort the list in presenter itself

**Comment by naman14** [comment]
@shaliniv16 PLEASE don't close existing PRs and create new ones with same changes. You can push more commits to existing PR by pushing more commits to the respective branch in your fork.

This allows everyone to have all the info about the PRs, the comments added, comments addressed, changes done etc. Community engagement is important, if you close PRs on which some changes were requested and create new ones that fix those changes but doesn't have the previous engagement, it defeats the purpose of how pull requests works.

If you need any help or are facing some problem with adding commits to a PR, feel free to ask here.

I see that you have deleted the forked repository so wouldn't be able to push changes to this PR but we will not accept changes that are done like this in the future.

**Comment by shaliniv16** [comment]
@naman14 The repository got deleted by mistake and I will ensure that I do not repeat this mistake in future. I apologize for the inconvenience caused due to the mistake.

**Pull request closed without merging**